### DISCUSSION TITLE
Show that advanced search filters are in use

### DISCUSSION THREAD

**Discussion by antonio8484** [snipe/snipe-it#9202] [Ideas]
Unless I'm missing something obvious, there should be a way to warn the user that advanced search is being used (and a 'clear filters' button would also be really useful). Sometimes I get only one or two lines under 'Assets' and it takes me a few seconds to understand that advanced search is filtering out results.

Even a red (vs grey) advanced search button, when filters are in place, could be helpful, although maybe a more obvious reminder (like a red, thick line or something) would do a better job.

**Comment by UserGoodtm** [comment]
I support the idea. Something like this is needed.
